packages/commons/logging-execa/src/loggingExeca.ts

```diff
@@ -37,5 +37,14 @@ export async function loggingExeca(
         command.stdout?.pipe(process.stdout);
         command.stderr?.pipe(process.stderr);
     }
-    return command;
+    return command.then((result) => {
+        // the execa types are incorrect and sometimes stdout and stderr and not defined
+        if (result.stdout == null) {
+            result.stdout = "";
+        }
+        if (result.stderr == null) {
+            result.stderr = "";
+        }
+        return result;
+    });
```

Comment:
this was the root cause of https://github.com/fern-api/fern/pull/4675, so updating the types here to prevent it from happening elsewhere